Sources/ScipioKit/Producer/PIF/ToolchainGenerator.swift

```diff
@@ -60,3 +60,36 @@ struct ToolchainGenerator {
         )
     }
 }
+
+fileprivate extension ToolchainGenerator {
+
+    /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`
+    /// Returns `macosx` sdk platform framework path.
```

Comment:
Please note the original implementation

```suggestion
    /// A non-caching environment-aware implementation of `SwiftSDK.sdkPlatformFrameworkPaths`
    /// This implementation is based on the original SwiftPM
    /// https://github.com/swiftlang/swift-package-manager/blob/release/6.0/Sources/PackageModel/SwiftSDKs/SwiftSDK.swift#L592-L595
    /// Returns `macosx` sdk platform framework path.
```